### DISCUSSION TITLE
cheatsheets/linux/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/cheatsheets#22] [General]
# cheatsheets/linux/

https://shanenull.com/cheatsheets/linux/

**Comment by shane0** [comment]
folders

**Comment by shane0** [comment]
https://youtu.be/HbgzrKJvDRw?si=ctINU-4HpcjjYtwP

**Reply by shane0** [reply]
thats about linux

**Comment by shane0** [comment]
50 cmds https://youtu.be/ZtqBQ68cfJc
